app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt

```diff
@@ -57,58 +68,101 @@ private fun BitwardenContentBlock(
     headerTextStyle: TextStyle = BitwardenTheme.typography.titleSmall,
     subtitleText: String? = null,
     subtitleTextStyle: TextStyle = BitwardenTheme.typography.bodyMedium,
+    showDivider: Boolean = true,
     @DrawableRes iconVectorResource: Int? = null,
     backgroundColor: Color = BitwardenTheme.colorScheme.background.secondary,
 ) {
-    Row(
-        modifier = modifier
-            .fillMaxWidth()
-            .background(backgroundColor),
-        verticalAlignment = Alignment.CenterVertically,
-    ) {
-        iconVectorResource
-            ?.let {
-                Spacer(Modifier.width(12.dp))
-                Icon(
-                    painter = rememberVectorPainter(it),
-                    contentDescription = null,
-                    tint = BitwardenTheme.colorScheme.icon.secondary,
-                    modifier = Modifier.size(24.dp),
-                )
-                Spacer(Modifier.width(12.dp))
+    var dividerStartPadding by remember { mutableStateOf(0.dp) }
+    val localDensity = LocalDensity.current
+    Column {
+        Row(
+            modifier = modifier
```

Comment:
The passed in modifier should always be on the outermost composable.

Reply:
To avoid the column completely you could always use the `bottomDivider` modifier extension.